### PR TITLE
OSDOCS-3550: Updated OCP mirror links for installations

### PR DESCRIPTION
**Pull request by mjpytlak**
Version(s):
CP to 4.11

Issue:
This PR addresses [osdocs-3550](https://issues.redhat.com/browse/OSDOCS-3550).

Link to docs preview:

- [Creating the RHCOS cluster image for the GCP infrastructure](http://file.rdu.redhat.com/mpytlak/osdocs-3550/installing/installing_gcp/installing-gcp-user-infra.html#installation-gcp-user-infra-rhcos_installing-gcp-user-infra)
- [The inventory.yml file](http://file.rdu.redhat.com/mpytlak/osdocs-3550/installing/installing_rhv/installing-rhv-user-infra.html#installation-rhv-about-inventory-yml_installing-rhv-user-infra)
- [Specifying the RHCOS image settings](http://file.rdu.redhat.com/mpytlak/osdocs-3550/installing/installing_rhv/installing-rhv-user-infra.html#installation-rhv-specifying-rhcos-image-settings_installing-rhv-user-infra)
- [Installing RHCOS by using an ISO image](http://file.rdu.redhat.com/mpytlak/osdocs-3550/installing/installing_bare_metal/installing-bare-metal.html#installation-user-infra-machines-iso_installing-bare-metal)
- [Installing RHCOS and starting the OpenShift Container Platform bootstrap process](http://file.rdu.redhat.com/mpytlak/osdocs-3550/installing/installing_vsphere/installing-vsphere.html#installation-vsphere-machines_installing-vsphere)

### CODE REVIEW

**Comment by mjpytlak** [comment]
@jstuever Several topics, including GCP content, reference https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.11/. This currently does not resolve, but looking to verify that this is in fact a valid URL that will resolve correctly at 4.11 GA.

CC: @rvanderp3 as the link is also referenced in vSphere doc.

**Comment by mjpytlak** [comment]
Confirmed with CoreOS team that the link to the mirror will resolve at 4.11 GA. [1]

https://coreos.slack.com/archives/C999USB0D/p1657654124440799

File: modules/installation-gcp-user-infra-rhcos.adoc [hunk 1]
```diff
@@ -12,9 +12,7 @@ your {product-title} nodes.
 .Procedure
 
 ifndef::openshift-origin[]
-. Obtain the {op-system} image from the
-link:https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.10/[{op-system} image mirror]
-page.
+. Obtain the {op-system} image from the link:https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.11/[{op-system} image mirror] page.
```
Comment (tmalove) [inline modules/installation-gcp-user-infra-rhcos.adoc:15]: Per your request in the channel, this link will resolve in 4.11 right?

**Review by tmalove — COMMENTED** [review]
LGTM!

**Comment by ahardin-rh** [comment]
LGTM!

**Pull request merged**